source/compute/compute-bp.rst

```diff
@@ -71,3 +71,25 @@ and balancing load amongst them) rather than vertically. It is possible to
 scale compute instances horizontally without downtime. Resizing compute
 instance vertically (up or down) will result in a brief downtime, because the
 operating system needs to reboot to pick up the new configuration.
+
+
+**************************
+Nested Virtualization
+**************************
+
+It is possible on the cloud to host virtual machines inside of an instance on
+the cloud. This feature is available from the default images that are provided
```

Comment:
Could we change the first sentence to "Nested virtualization is a feature that allows you to run KVM virtual machine inside of a nova instance"?